package.json

```diff
@@ -46,6 +46,7 @@
   },
   "enableProposedApi": true,
   "activationEvents": [
+    "*",
```

Comment:
Why do you want to activate the extension unconditionally now? It will mean that even if I launch VS Code to view or edit a .txt file our extension will start. Probably not much of an overhead, but it should still be justifiable.

And if we **do** adopt this I suggest all the other entries should be removed because they serve no purpose.

Reply:
It looks like it's only one way to add a panel to the status bar, for some empty folder.

Reply:
I prefer the current behaviour. Opening an empty folder doesn't add ObjectScript panel to the status bar but **does** add the ObjectScript item to the activity bar. Clicking that icon activates the extension, so the status bar panel now appears. The empty ObjectScript view also gives the user some help about what to do next.